packages/extension-chakra-storefront/src/pages/cart/partials/empty-cart.jsx

```diff
@@ -45,15 +45,14 @@ const EmptyCart = ({isRegistered}) => {
                             </Text>
                         </Stack>
                         <Stack justify="center" direction={['column', 'row']} gap={4}>
-                            <Button variant="outline" asChild>
-                                <Link
-                                    borderColor="blue.600"
-                                    href={'/'}
-                                    bg="transparent"
-                                    width={['343px', '220px']}
-                                    variant={isRegistered ? 'solid' : 'outline'}
-                                    color={isRegistered ? 'white' : 'blue.600'}
-                                >
+                            <Button
+                                color={isRegistered ? 'white' : 'blue.600'}
+                                variant={isRegistered ? 'solid' : 'outline'}
+                                borderColor="blue.600"
+                                width={['343px', '220px']}
+                                asChild
```

Comment:
fix empty cart on registered user screen
![image](https://github.com/user-attachments/assets/ca9b3fba-04c3-44de-8ebb-4eba9297eae6)

Reply:
it is fixed. Good catch